.pypirc

```diff
@@ -0,0 +1,16 @@
+# This is template for .pypirc file
```

Comment:
I don't think we should include this file.

Reply:
removed